package.json

```diff
@@ -23,55 +23,55 @@
   "dependencies": {
     "@expo-google-fonts/roboto": "0.2.3",
     "@expo-google-fonts/rubik": "0.2.3",
-    "@expo/cli": "^0.10.11",
+    "@expo/webpack-config": "^19.0.0",
     "@ory/client": "1.1.46",
     "@ory/themes": "0.0.21",
     "@react-native-async-storage/async-storage": "1.18.2",
-    "@react-native-community/async-storage": "^1.12.1",
-    "@react-native-community/masked-view": "^0.1.11",
-    "@react-navigation/native": "~6.1.7",
-    "@react-navigation/stack": "~6.3.17",
-    "@sentry/integrations": "7.64.0",
+    "@react-native-community/async-storage": "1.12.1",
+    "@react-native-community/masked-view": "0.1.11",
+    "@react-navigation/native": "6.1.7",
+    "@react-navigation/stack": "6.3.17",
+    "@sentry/integrations": "7.52.0",
     "@sentry/react-native": "5.5.0",
-    "@types/node": "~18.17.5",
-    "axios": "~1.4.0",
-    "expo": "^49.0.0",
-    "expo-application": "~5.3.0",
-    "expo-auth-session": "~5.0.2",
-    "expo-cli": "~6.0.6",
-    "expo-constants": "~14.4.2",
-    "expo-device": "~5.4.0",
-    "expo-linking": "~5.0.2",
-    "expo-random": "~13.2.0",
-    "expo-secure-store": "~12.3.1",
-    "expo-status-bar": "~1.6.0",
-    "expo-updates": "~0.18.11",
-    "expo-web-browser": "~12.3.2",
+    "@types/node": "18.17.5",
+    "axios": "1.4.0",
+    "expo": "49.0.10",
+    "expo-application": "5.3.0",
+    "expo-auth-session": "5.0.2",
+    "expo-cli": "6.3.10",
```

Comment:
AFAIK you don't need this package anymore. It's bundled into the `expo` pacakge. https://blog.expo.dev/the-new-expo-cli-f4250d8e3421

Reply:
I removed it like 3 times, but something kept adding it back??? Thanks, will do.